migrate_pim/apply_patch/apply_patch_ee_flexibility.rst

```diff
@@ -27,9 +27,9 @@ Run the composer update command:
 
 Be aware that your composer.json won't be updated and some dependencies might be missing or from an outdated version.
 
-You have to check whether the latest composer.json is different from your own. In this case you should backup your current composer.json and download the newest one beforehand.
+You have to make sure you have the latest version of composer.json. If this isn't the case, you should backup your current composer.json and download the newest one before updating.
```

Comment:
How do they know if they have the latest version of `composer.json`? Perhaps we should just say that they should always download the newest version to be safe?

If they are relying on other bundles, they should keep track of these and make sure they are adding them.

Reply:
I've added a Troubleshooting section that details where to get the composer.json and how to replace older versions